packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -14,174 +14,96 @@
-  isPluginApplicableToEntity as isPullRequestsAvailable,
-  PullRequestsStatsCard,
-  Router as PullRequestsRouter,
-} from '@roadiehq/backstage-plugin-github-pull-requests';
-import {
-  isPluginApplicableToEntity as isTravisCIAvailable,
-  RecentTravisCIBuildsWidget,
-  Router as TravisCIRouter,
-} from '@roadiehq/backstage-plugin-travis-ci';
-import React, { ReactNode, useMemo, useState } from 'react';
-import BadgeIcon from '@material-ui/icons/CallToAction';
-
-export const CICDSwitcher = ({ entity }: { entity: Entity }) => {
-  // This component is just an example of how you can implement your company's logic in entity page.
-  // You can for example enforce that all components of type 'service' should use GitHubActions
-  switch (true) {
-    case isJenkinsAvailable(entity):
-      return <JenkinsRouter entity={entity} />;
-    case isBuildkiteAvailable(entity):
-      return <BuildkiteRouter entity={entity} />;
-    case isCircleCIAvailable(entity):
-      return <CircleCIRouter entity={entity} />;
-    case isCloudbuildAvailable(entity):
-      return <CloudbuildRouter entity={entity} />;
-    case isTravisCIAvailable(entity):
-      return <TravisCIRouter entity={entity} />;
-    case isGitHubActionsAvailable(entity):
-      return <GitHubActionsRouter entity={entity} />;
-    default:
-      return (
-        <EmptyState
-          title="No CI/CD available for this entity"
-          missing="info"
-          description="You need to add an annotation to your component if you want to enable CI/CD for it. You can read more about annotations in Backstage by clicking the button below."
-          action={
-            <Button
-              variant="contained"
-              color="primary"
-              href="https://backstage.io/docs/features/software-catalog/well-known-annotations"
-            >
-              Read more
-            </Button>
-          }
-        />
-      );
-  }
-};
-
-const RecentCICDRunsSwitcher = ({ entity }: { entity: Entity }) => {
-  let content: ReactNode;
-  switch (true) {
-    case isJenkinsAvailable(entity):
-      content = <JenkinsLatestRunCard branch="master" variant="gridItem" />;
-      break;
-    case isTravisCIAvailable(entity):
-      content = <RecentTravisCIBuildsWidget entity={entity} />;
-      break;
-    case isGitHubActionsAvailable(entity):
-      content = (
-        <RecentWorkflowRunsCard entity={entity} limit={4} variant="gridItem" />
-      );
-      break;
-    default:
-      content = null;
-  }
-  if (!content) {
-    return null;
-  }
-  return (
-    <Grid item sm={6}>
-      {content}
-    </Grid>
-  );
-};
-
-export const ErrorsSwitcher = ({ entity }: { entity: Entity }) => {
-  switch (true) {
-    case isRollbarAvailable(entity):
-      return <RollbarRouter entity={entity} />;
-    default:
-      return <SentryRouter entity={entity} />;
-  }
-};
+// import {
+//   EntityBuildkiteContent,
+//   isBuildkiteAvailable,
+// } from '@roadiehq/backstage-plugin-buildkite';
```

Comment:
Commented out the plugins that can't be used yet, and then let's add them back as they become available

Reply:
cc @dtuite did you already see this?

Reply:
@Fox32 Nope. Just seeing it now. We will get this fixed ASAP.

Reply:
@Rugvip Can you elaborate what the issue on buildkite plugin is preventing it from being used?

Reply:
I created some issues around these, https://github.com/RoadieHQ/backstage-plugin-travis-ci/issues/52, and https://github.com/RoadieHQ/backstage-plugin-github-insights/issues/125. One is fixed but the other one is still open, and I believe it applies to all of the plugins.

I think the best way to check what's needed is to try out this branch and uncomment the code, iirc it's what is expected to work. except that the `is*Available` return value should not need to be case to a bool, so fix for that would be nice too :grin: